pynamodb/models.py

```diff
@@ -1261,10 +1261,21 @@ def _get_connection(cls):
         """
         Returns a (cached) connection
         """
-        if not hasattr(cls, "Meta") or cls.Meta.table_name is None:
+        if not hasattr(cls, "Meta"):
             raise AttributeError(
-                """As of v1.0 PynamoDB Models require a `Meta` class.
-                See https://pynamodb.readthedocs.io/en/latest/release_notes.html"""
+                'As of v1.0 PynamoDB Models require a `Meta` class.\n'
+                'Model: {}.{}\n'
+                'See https://pynamodb.readthedocs.io/en/latest/release_notes.html\n'.format(
+                    cls.__module__, cls.__name__,
+                ),
+            )
+        elif cls.Meta.table_name is None:
```

Comment:
```elif not hasattr(cls.Meta, "table_name") or cls.Meta.table_name is None:```

in case `Meta` doesn't have a `table_name`

Reply:
While I agree: this isn't a regression here. Do you still want me to change that?

Reply:
I agree the current behaviour is errorness, but it would be nice to have this corrected as well. Appreciated!

Reply:
done!

Reply:
thanks!